js/app/dashboard/components/Tour.js

```diff
@@ -91,7 +92,11 @@ const RenderEditNameForm = ({children, tour, isLoading}) => {
 }
 
 
-const Tour = ({ tour, draggableIndex }) => {
+const Tour = ({ tour, tourId, draggableIndex }) => {
+
+  if (!tour && tourId) {
+    tour = useSelector(state => selectTourById(state, tourId))
```

Comment:
Hooks should not be called conditionally: https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level